h2o-assembly/build.gradle

```diff
@@ -10,6 +10,7 @@ dependencies {
   if (project.hasProperty("doIncludeOrc") && project.doIncludeOrc == "true") {
     compile project(":h2o-orc-parser")
   }
+  compile project(":h2o-parquet-parser")
```

Comment:
Oki, so at the end the resulting h2o.jar assembly will get `org.apache.hadoop:hadoop-common:2.6.0-cdh5.4.0` version of hadoop